test/new-e2e/pkg/runner/parameters/store_config_file.go

```diff
@@ -34,8 +34,9 @@ type Config struct {
 
 // ConfigParams instance contains config relayed parameters
 type ConfigParams struct {
-	AWS   AWS   `yaml:"aws"`
-	Agent Agent `yaml:"agent"`
+	AWS       AWS    `yaml:"aws"`
+	Agent     Agent  `yaml:"agent"`
+	OutputDir string `yaml:"outputDir"`
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Could you update it also in the setup script in `test-infra` ? https://github.com/DataDog/test-infra-definitions/blob/main/tasks/config.py

Reply:
https://github.com/DataDog/test-infra-definitions/pull/561